### PR TITLE
Document --cli-script introduced by WFCORE-5324

### PR DESCRIPTION
**Pull request by jfdenise**
Document new argument that https://issues.redhat.com/browse/WFCORE-5324 will bring.

### CODE REVIEW

File: docs/res
Comment (bstansberry) [inline docs/res:1]: This file looks like an unintended inclusion.

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: docs/res
Comment (jfdenise) [reply docs/res]: Thanks, removed.

**Comment by jmesnil** [comment]
wildfly/wildfly-core#4504 has been merged. This PR can be merged after the wildlfy-core upgrade PR is merged

**Pull request merged**
